packages/moonstone/VideoPlayer/VideoPlayer.js

```diff
@@ -601,6 +601,8 @@ const VideoPlayerBase = class extends React.Component {
 		on('keypress', this.activityDetected);
 		on('keydown', this.handleGlobalKeyDown);
 		this.startDelayedFeedbackHide();
+		this.startDelayedTitleHide();
+		this.startAutoCloseTimeout();
```

Comment:
Is it possible to start the timer when controls are actually visible instead of starting at mounting time? It just seems a bit unnecessary.